### DISCUSSION TITLE
Problems after the update V1.122.1

### DISCUSSION THREAD

**Discussion by wiedaar** [immich-app/immich#14567] [Q&A]
I have no idea what mine logs  where saying because i tried to restore immich from the back-up but that did'nt work, a lot of error messages after a while in windows command manager. Then i threw everything away and installed immich again only from the docker-compose.yml en the .env file, wich worked fine. that is until today, now i got this error with my external database, my photos are on a different drive :
2024-12-07 21:26:21 immich_server            | [Nest] 7  - 12/07/2024, 6:26:21 PM    WARN [Microservices:LibraryService] No valid import paths found for library bb

the .env has not changed and now it does'nt want to read the drive

this is in the .env file 
EXTERNAL_PATH=D:/fotos en videos

So, hours of naming everybody in the photo's are gone

**Comment by alextran1502** [comment]
Can you provide docker compsoe and .env file?

**Reply by wiedaar** [reply]
Docker-Compose.yml 

name: immich

services:
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
   

    volumes:
      - ${EXTERNAL_PATH}:/usr/src/app/external
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime
    env_file:
      - .env
    ports:
      - 2283:2283
    depends_on:
      - redis
      - database
    restart: always

.env

# You can find documentation for all the supported env variables at https://immich.app/docs/install/environment-variables

# The location where your uploaded files are stored
UPLOAD_LOCATION=C:/docker/Immich/library
# The location where your database files are stored
DB_DATA_LOCATION=C:/docker/Immich/postgres

TZ="Etc/GMT+2"

EXTERNAL_PATH=D:/fotos en videos

# The Immich version to use. You can pin this to a specific version like "v1.71.0"
IMMICH_VERSION=release

# Connection secret for postgres. You should change it to a random password
DB_PASSWORD=secret

# The values below this line do not need to be changed
###################################################################################
DB_USERNAME=postgres
DB_DATABASE_NAME=immich

**Reply by alextran1502** [reply]
Can you post a screenshot of the library setting path in Immich web?

**Comment by wiedaar** [comment]
Where can i find that?

I installed Photoprims just to see if it scans my D: drive and it does that without a problem.



Van: Alex ***@***.***>
Verzonden: zaterdag 7 december 2024 23:05
Aan: immich-app/immich ***@***.***>
CC: wiedaar ***@***.***>; Author ***@***.***>
Onderwerp: Re: [immich-app/immich] Problems after the update V1.122.1 (Discussion #14567)


Can you post a screenshot of the library setting path in Immich web?

-

**Comment by wiedaar** [comment]
I have tried this :

EXTERNAL_PATH=D:/fotos en videos
EXTERNAL_PATH="D:/fotos en videos"
EXTERNAL_PATH=./D:/fotos en videos/

Non of them works
Immich starts without problems only when i start the scanning of the external database i got this error :

2024-12-08 10:50:27 immich_server            | [Nest] 7  - 12/08/2024, 7:50:27 AM    WARN [Microservices:LibraryService] No valid import paths found for library f5b57488-10b4-4632-9b06-545488a486e5

The problems started with the 2 latest updates
Van: Alex ***@***.***>
Verzonden: zaterdag 7 december 2024 23:05
Aan: immich-app/immich ***@***.***>
CC: wiedaar ***@***.***>; Author ***@***.***>
Onderwerp: Re: [immich-app/immich] Problems after the update V1.122.1 (Discussion #14567)


Can you post a screenshot of the library setting path in Immich web?

-

**Comment by wiedaar** [comment]
/usr/src/app/external

Forgot about that piece 😉


Van: Alex ***@***.***>
Verzonden: zaterdag 7 december 2024 23:05
Aan: immich-app/immich ***@***.***>
CC: wiedaar ***@***.***>; Author ***@***.***>
Onderwerp: Re: [immich-app/immich] Problems after the update V1.122.1 (Discussion #14567)


Can you post a screenshot of the library setting path in Immich web?

**Comment by alextran1502** [comment]
For sanity, you can rename the folder to "fotos_en_videos" basically without white space. Then your EXTERNAL_PATH will be "D:\fotos_en_videos"

**Comment by wiedaar** [comment]
Immich is running again, i already started with giving names 😊
I have installed the update and nothing has crashed 😉

I did see a new fearure that you can scan for faces per foto?

I was looking at photoprism but delete it after it was finished with scanning, what a bad face recognition

Van: Alex ***@***.***>
Verzonden: zondag 8 december 2024 22:44
Aan: immich-app/immich ***@***.***>
CC: wiedaar ***@***.***>; Author ***@***.***>
Onderwerp: Re: [immich-app/immich] Problems after the update V1.122.1 (Discussion #14567)


For sanity, you can rename the folder to "fotos_en_videos" basically without white space. Then your EXTERNAL_PATH will be "D:\fotos_en_videos"
